web/src/components/Front.jsx

```diff
@@ -5,30 +5,41 @@ import { SafeAreaView } from "react-native-safe-area-context";
 import { ROUTES } from "../constants";
 import ItemCards from "./ItemCards";
 import ItemDescript from "./ItemDescript";
+import { createDrawerNavigator } from "@react-navigation/drawer";
 
 const Front = () => {
-  const Stack = createStackNavigator();
+  // const Stack = createStackNavigator();
+  const Drawer = createDrawerNavigator();
 
   return (
     <>
       <SafeAreaView>
-        <Text>This is gonna be the Header Bar</Text>
+        {/* <Text>This is gonna be the Header Bar</Text>
         <Text>
           This is a nested navigator. Add drawer navigators into Front.jsx
```

Comment:
Actually since we don't have the hamburger ready on the header bar, could you make a simple button or text to toggle the drawer open and close for demo purposes?

<Button title="Toggle drawer" onPress={() => navigation.toggleDrawer()} />